app/components/base/Table.tsx

```diff
@@ -5,7 +5,7 @@ import { HEADLINE_TEXT, PARAGRAPH_TEXT, GREY_1, GREY_2 } from "../colors";
 import { breakUp } from "../breakpoints";
 
 // Styles for Table
-const StyledTable = styled.div`
```

Comment:
When MDX (and React/Next) hydrate a `<table>` element, this line replaces it with `<div>` with some additional styling added to it. This causes a hydration error since `<thead>`, `<tbody>`, etc. are not legal children of `<div>` but rather only `<table>`.